Bottom: 2be9e3761853ac744e0278a909de977659590fc9
Top:    6b76e046589474049f277da016c5013bb08c4921
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-02-01 16:20:07 -0600

Fix .ssh folder permission

The start.sh scripts were mistakenly changing the permissions of
$HOME/.ssh to 0600, removing the execute bit for the user and 
effectively removing access.


---
diff --git a/docker/buildbot-worker-ptxdist/buildbot/start.sh b/docker/buildbot-worker-ptxdist/buildbot/start.sh
index 0299c58..23ce583 100755
--- a/docker/buildbot-worker-ptxdist/buildbot/start.sh
+++ b/docker/buildbot-worker-ptxdist/buildbot/start.sh
@@ -2,7 +2,7 @@
 
 mkdir -p $HOME/.ssh
 cp -R /run/secrets/ssh/* $HOME/.ssh
-chmod -R 600 $HOME/.ssh
+chmod -R 600 $HOME/.ssh/*
 
 ln -v -s -f /home/buildbot/buildbot.tac /buildbot/buildbot.tac
 exec twistd --nodaemon --logfile=- --pidfile=/tmp/twistd.pid --python=buildbot.tac
